de.fu_berlin.inf.dpp.whiteboard/frontend/.eslintrc.json

```diff
@@ -20,7 +20,7 @@
   ],
   "settings": {
     "react": {
-      "version": "detect"
+      "version": "latest"
```

Comment:
What does "latest" mean? Does it mean that always the eslint rules of the latest react release are used (even if we don't use the latest version)?

Reply:
this change will fit more likely to another patch that I will be adding soon.
I will remove this now
But yes "latest" would mean latest react version even if were are not using the latest one.